v4/source/system.yaml

```diff
@@ -578,6 +578,14 @@
         Must have `manage_system` permission.
 
         __Minimum server version__: 5.20
+
+        ##### Note
+
+        The Plugins are stored as a map, and since a map may recursively go 
+        down to any depth, individual fields of a map are not changed. 
+        Avoid adding more complexity by attempting to support 
+        patch on a map by using the `update config` 
+        (PUT api/v4/config) endpoint as a workaround.
```

Comment:
Consider using the `update config` (PUT api/v4/config) endpoint
to update a plugins configurations.